builtin/plugins/address_mapper/address_mapper.go

```diff
@@ -68,8 +65,7 @@ func (am *AddressMapper) Init(ctx contract.Context, req *InitRequest) error {
 	return nil
 }
 
-// AddIdentityMapping adds a mapping between a DAppChain account and a Mainnet account.
-// The caller must provide proof of ownership of the Mainnet account.
+// AddIdentityMapping adds a mapping between a Loom account and a foreign account. The caller must provide proof of ownership of the foreign account.
```

Comment:
@enlight Quick question: is this how we should name them ("Loom account" and "foreign account")?

Reply:
Should probably use more consistent naming... local account and foreign account.

Reply:
@enlight Gotcha, thanks!